maestro/server/helm/templates/maestro.deployment.yaml

```diff
@@ -88,7 +89,7 @@ spec:
         - --db-user-file=/secrets/db/db.user
         - --db-password-file=/secrets/db/db.password
         - --db-name-file=/secrets/db/db.name
-        - --db-rootcert=/secrets/db/db.ca_cert
```

Comment:
the file `/secrets/db/db.ca_cert` doesn't exist causing issue when the DB connection string is parsed.
Leaving this to empty will cause the system root cert to be used.